lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1127,6 +1182,39 @@ def test_expand_public_user(self):
         )
         self.verify_expanded_public_user(result['membership'][0]['user'])
 
+    @ddt.unpack
+    @ddt.data(
+        ('student_enrolled', 200),
+        ('student_masters_not_on_team', 200),
+        ('student_masters', 200),
+        ('staff', 200)
+    )
+    def test_bubble_protection(self, requesting_user, expected_response):
+        team = self.get_team_detail(
```

Comment:
student enrolled should be getting a 404, he shouldn't be able to know that this team exists in any way